src/main/java/org/opensearch/security/privileges/PrivilegesEvaluationContext.java

```diff
@@ -45,7 +46,7 @@ public class PrivilegesEvaluationContext {
     private final IndexResolverReplacer indexResolverReplacer;
     private final IndexNameExpressionResolver indexNameExpressionResolver;
     private final Supplier<ClusterState> clusterStateSupplier;
-
+    private final Permissions permissionsForApiToken;
```

Comment:
nit: just call this `permissions`. API Tokens may directly map to permissions, but this could be used generically for any such similar use case.

Reply:
Done - good idea to keep it general to support more use cases in the future.